render/persistentvolume.go

```diff
@@ -116,3 +120,82 @@ func (v pvTocontrollerRenderer) Render(rpt report.Report) Nodes {
 	}
 	return Nodes{Nodes: nodes}
 }
+
+// SCToSPCRenderer is a Renderer which produces a renderable SPC Disk
+var SCToSPCRenderer = scTospcRenderer{}
+
+//scTospcRenderer is a Renderer to render SC & SPC.
+type scTospcRenderer struct{}
+
+//Render renders the SC & SPC nodes with adjacency.
+func (v scTospcRenderer) Render(rpt report.Report) Nodes {
+	nodes := make(report.Nodes)
+	for spcID, spcNode := range rpt.StoragePoolClaim.Nodes {
+		storagePoolClaimName, _ := spcNode.Latest.Lookup(kubernetes.Name)
+		spcname := strconv.Quote(storagePoolClaimName)
+		for scID, scNode := range rpt.StorageClass.Nodes {
+			storageClass, _ := scNode.Latest.Lookup(kubernetes.Value)
+			if strings.Contains(storageClass, ":") {
+				storagevalue := strings.Split(storageClass, ":")
```

Comment:
Instead of splitting from `:`, can we split it from `"`?
This will reduce the effort of adding quotes into spcName
`spcname := strconv.Quote(storagePoolClaimName)`

for splitting from `"` you can use `strings.Split(storageClass, "\"")`